logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SlowSyncWorker.kt

```diff
@@ -18,6 +19,7 @@ class SlowSyncWorker(
         kaliumLogger.d("Sync: Starting SlowSync")
 
         val result = try{
+            userSessionScope.getRemoteFeatureConfigsStatusAndPersist
```

Comment:
Also, could you please add a TODO to move the feature flags stuff out of the SlowSync later? I'll explain in the review comment.